frontend/src/component/project/Project/ProjectFeatureToggles/FeatureToggleSwitch/FeatureToggleSwitch.tsx

```diff
@@ -113,9 +122,9 @@ export const FeatureToggleSwitch: VFC<IFeatureToggleSwitchProps> = ({
         }
     };
 
-    const onClick = async (e: React.MouseEvent) => {
+    const handleClick = async () => {
```

Comment:
with onClick and handleClick it's difficult to tell how they differ by looking at the names